build.cmd

```diff
@@ -18,6 +18,12 @@ if %errorlevel% equ 0 (
   set outloop=true
 )
 
+echo %* | findstr /i /C:"FloatingTestRuntimeDependencies=true"  1>nul
```

Comment:
For easy maintenance, I'd suggest we keep our build scripts consistent with corefx build scripts if possible.    Corefx build scripts uses <ExpectedPrerelease>rc3-23729</ExpectedPrerelease>.

I agree corefx build scripts are not always best practice. But easy maintenance weighs more here, I think.